src/composables/useShare.ts

```diff
@@ -0,0 +1,60 @@
+import { useI18n } from 'vue-i18n';
+import { useClipboard } from '@vueuse/core';
```

Comment:
Would be great to use that in the others place where we use clipboard. And also add a notification to confirm copy worked.